config.lua

```diff
@@ -6,6 +6,7 @@ Config.WipeInventoryOnRespawn = true -- Enable or disable removing all the playe
 Config.Helicopter = "polmav" -- Helicopter model that players with the ambulance job can use
 Config.BillCost = 2000 -- Price that players are charged for using the hospital check-in system
 Config.DeathTime = 300 -- How long the timer is for players to bleed out completely and respawn at the hospital
+Config.DeathTime = 45 -- How long the timer is for players to bleed out completely and respawn at the hospital
```

Comment:
We do not want to add this Duplicate config item.

Reply:
Thank you for pointing this out not sure why i had a second variable but i will fix that tonight